.github/workflows/publish-release.yml

```diff
@@ -8,6 +8,9 @@ on:
 jobs:
   call_workflow:
     name: Run Release Workflow
+    env:
+      nexusUser: ${{ secrets.NEXUS_USERNAME }}
+      nexusPassword: ${{ secrets.NEXUS_PASSWORD }}
```

Comment:
You cannot add envs here. Have to add them as secrets. They will be added as envs by the workflow.

Reply:
Updated in [bed03a2](https://github.com/ballerina-platform/module-ballerinax-persist.sql/pull/47/commits/bed03a2b47cbc78899961c5c6ec867f95ea4a610)